src/main/public/main-dev.js

```diff
@@ -188,7 +188,7 @@ eval("\nObject.defineProperty(exports, \"__esModule\", ({ value: true }));\ncons
exports.STATEMENT_OF_TRUTH = exports.CHECK_YOUR_ANSWERS = exports.CONTACT_PREFERENCES = exports.UPLOAD_YOUR_DOCUMENTS = exports.PAY_YOUR_FEE = exports.CONTACT_DETAILS = exports.MANUAL_ADDRESS = exports.SUBJECT_CONTACT_DETAILS = exports.SELECT_ADDRESS = exports.FIND_ADDRESS = exports.DATE_OF_BIRTH = exports.SUBJECT_DETAILS = exports.USER_ROLE = exports.FULL_NAME = exports.EDGE_CASE_URL = exports.APPLYING_WITH_URL = exports.TASK_LIST_URL = exports.ADDITIONAL_DOCUMENTS_UPLOAD = exports.DOCUMENT_UPLOAD_URL = exports.CSRF_TOKEN_ERROR_URL = exports.KEEP_ALIVE_URL = exports.TIMED_OUT_URL = exports.SAVE_AND_SIGN_OUT = exports.SIGN_OUT_URL = exports.SIGN_IN_URL = exports.CALLBACK_URL = exports.HOME_URL = void 0;\nexports.HOME_URL = '/';\nexports.CALLBACK_URL = '/receiver';\nexports.SIGN_IN_URL = '/login';\nexports.SIGN_OUT_URL = '/logout';\nexports.SAVE_AND_SIGN_OUT = '/save-and-sign-out';\nexports.TIMED_OUT_URL = '/timed-out';\nexports.KEEP_ALIVE_URL = '/keep-alive';\nexports.CSRF_TOKEN_ERROR_URL = '/csrf-token-error';\nexports.DOCUMENT_UPLOAD_URL = '/document-manager';\nexports.ADDITIONAL_DOCUMENTS_UPLOAD = '/upload-additional-documents';\nexports.TASK_LIST_URL = '/task-list';\nexports.APPLYING_WITH_URL = '/applying-with';\nexports.EDGE_CASE_URL = '/edge-case';\nexports.FULL_NAME = '/full-name';\nexports.USER_ROLE = '/user-role';\nexports.SUBJECT_DETAILS = '/subject-details';\nexports.DATE_OF_BIRTH = '/date-of-birth';\nexports.FIND_ADDRESS = '/address/lookup';\nexports.SELECT_ADDRESS = '/address/select';\nexports.SUBJECT_CONTACT_DETAILS = '/subject-contact-details';\nexports.MANUAL_ADDRESS = '/address/manual';\nexports.CONTACT_DETAILS = '/contact-details';\nexports.PAY_YOUR_FEE = '/payment/pay-your-fee';\nexports.UPLOAD_YOUR_DOCUMENTS = '/upload-your-documents';\nexports.CONTACT_PREFERENCES = '/contact-preferences';\nexports.CHECK_YOUR_ANSWERS = '/check-your-answers';\nexports.STATEMENT_OF_TRUTH = '/statement-of-truth';\nexports.APPLICATION_SUBMITTED = '/application-submitted';\nexports.COOKIES = '/cookies';\nexports.PRIVACY_POLICY = '/privacy-policy';\nexports.ACCESSIBILITY_STATEMENT = '/accessibility-statement';\nexports.TERMS_AND_CONDITIONS = '/terms-and-conditions';\nexports.CONTACT_US = '/contact-us';\nexports.REPRESENTATION = '/representation';\nexports.REPRESENTATION_QUALIFIED = '/representation-qualified';\nexports.REPRESENTATIVES_DETAILS = '/representative-details';\nexports.CHECK_JURISDICTION = '/check-jurisdiction';\nexports.WHERE_YOUR_LIVES_ARE_BASED_URL = '/where-your-lives-are-based';\nexports.JURISDICTION_INTERSTITIAL_URL = '/you-can-use-english-welsh-courts';\nexports.RESIDUAL_JURISDICTION = '/are-you-eligible-for-residual-jurisdiction';\nexports.JURISDICTION_MAY_NOT_BE_ABLE_TO = '/you-may-not-be-able-to-england-and-wales';\nexports.JURISDICTION_CONNECTION_SUMMARY = '/connection-summary';\nexports.PAY_AND_SUBMIT = '/pay-and-submit';\nexports.ELIGIBILITY_URL = '/eligibility';\n\n\n//# sourceURL=webpack://ds-ui/./src/main/steps/urls.ts?");
```

Comment:
Don't need this change.  I haven't worked out how to prevent this file being changed automatically!